crates/test-utils/src/fixture.rs

```diff
@@ -180,7 +180,9 @@ impl Fixture {
         let mut cfg_key_values = Vec::new();
         let mut env = FxHashMap::default();
         let mut introduce_new_source_root = None;
-        let mut target_data_layout = None;
+        let mut target_data_layout = Some(
```

Comment:
Nit: some of the tests already have a layout annotation, should we also set it globally here?

Reply:
Currently all of const eval tests, and some of tests in infer need it, so I think it make sense to have it for default. If we keep this, those annotations can be removed.

Reply:
My sight preference would be to make them explicit, since it seems like something that might catch people off-guard. But it's not very important tbh.

Reply:
Your point is valid, but I think there are too much tests that needs it (and there will be more) so the visual noise doesn't worth it, and also keep them all in a same place makes future changes to it easier. So I would prefer it to be present by default.

Reply:
Ye I think keeping a default here is fine. No need to remove the setting of this though, maybe we do want to specifically test different layouts in some places at some point